components/test_util/src/logging.rs

```diff
@@ -96,8 +101,12 @@ pub fn init_log_for_test() {
     )
     .unwrap();
     let writer = output.map(|f| Mutex::new(File::create(f).unwrap()));
-    // we don't mind set it multiple times.
-    let drainer = CaseTraceLogger { f: writer };
+    // We don't mind set it multiple times.
+    // We hardly ever read rocksdb log in tests.
+    let drainer = CaseTraceLogger {
+        f: writer,
+        skip_tags: vec!["rocksdb_log".to_owned(), "raftdb_log".to_owned()],
+    };
 
     // Default disabled log targets for test.
     let disabled_targets = vec!["tokio_core".to_owned(), "tokio_reactor".to_owned()];
```

Comment:
can we disable the log it by `disabled_targets`?

Reply:
`target` and `tags` are different parts of a log record.

Reply:
yes, but it can be filtered the rocksdb log by target, we don't need to add an extra mechanism. And `disabled_targets` is more flexible which can be set by env variable.

Reply:
How?

Reply:
I checked the output of logs, the full module name is "engine_rocks::logger". I prefer not to filter by module name as it doesn't necessary only mean logs from rust-rocksdb. Besides, `skip_tags` only works in tests.

Reply:
alright